chainercv/links/model/ssd/transforms.py

```diff
@@ -0,0 +1,289 @@
+    else:
+        cv_img = saturation(cv_img)
+        cv_img = hue(cv_img)
+        cv_img = contrast(cv_img)
+
+    return cv_img.astype(np.float32).transpose(2, 0, 1)[::-1]
+
+
+def random_crop_with_bbox_constraints(
+        img, bbox, min_scale=0.3, max_scale=1,
+        max_aspect_ratio=2, constraints=None,
+        max_trial=50, return_param=False):
+    """Crop an image randomly with bounding box constraints.
+
+    This data augmentation is used in training of
+    Single Shot Multibox Detector [#]_. More details can be found in
+    data augmentation section of the original paper.
+
+    .. [#] Wei Liu, Dragomir Anguelov, Dumitru Erhan, Christian Szegedy,
+       Scott Reed, Cheng-Yang Fu, Alexander C. Berg.
+       SSD: Single Shot MultiBox Detector. ECCV 2016.
+
+    Args:
+        img (~numpy.ndarray): An image array to be cropped. This is in
+            CHW format.
+        bbox (~numpy.ndarray): Bounding boxes used for constraints.
+            The shape is :math:`(R, 4)`.
+            :math:`R` is the number of bounding boxes.
+        min_scale (float): The minimum ratio between a cropped
+            region and the original image. The default value is :obj:`0.3`.
+        max_scale (float): The maximum ratio between a cropped
+            region and the original image. The default value is :obj:`1`.
+        max_aspect_ratio (float): The maximum aspect ratio of cropped region.
+            The default value is :obj:`2`.
+        constaraints (iterable of tuples): An iterable of constraints.
+            Each constraint should be :obj:`(min_iou, max_iou)` format.
+            If you set :obj:`min_iou` or :obj:`max_iou` to :obj:`None`,
+            it means not limited.
+            If this argument is not specified, :obj:`((0.1, None), (0.3, None),
+            (0.5, None), (0.7, None), (0.9, None), (None, 1))` will be used.
+        max_trial (int): The maximum number of trials to be conducted
+            for each constraint. If this function
+            can not find any region that satisfies the constraint in
+            :math:`max\_trial` trials, this function skips the constraint.
+            The default value is :obj:`50`.
+        return_param (bool): If :obj:`True`, this function returns
+            information of intermediate values.
+
+    Returns:
+        ~numpy.ndarray or (~numpy.ndarray, dict):
+
+        If :obj:`return_param = False`,
+        returns an array :obj:`img` that is cropped from the input
+        array.
+
+        If :obj:`return_param = True`,
+        returns a tuple whose elements are :obj:`img, param`.
+        :obj:`param` is a dictionary of intermediate parameters whose
+        contents are listed below with key, value-type and the description
+        of the value.
+
+        * **constraint** (*tuple*): The chosen constraint.
+        * **y_slice** (*slice*): A slice used to crop the input image.\
+            The relation below holds together with :obj:`x_slice`.
```

Comment:
There is no relation below. I think you should add the relation or remove this sentence.